src/main/resources/assets/waila/lang/de_DE.lang

```diff
@@ -12,14 +13,18 @@ screen.button.hidden=Versteckt
 screen.button.visible=Sichtbar
 screen.button.maintained=Gewartet
 screen.button.toggled=An/Aus
+screen.hwyla.warning=§cWarnung:§r Du verwendest eine Abspaltung(fork) von Waila, auch genannt Hwyla. Du kannst die (zum größtenteils fertige) Liste der Änderungen sehen, wenn du auf die Schaltfläche oben links in diesem Fenster klickst.
 
 choice.showhidewaila=Zeige/Verstecke Waila
 choice.showhideidmeta=Zeige ID:Metadaten
 choice.toggledmaintained=An/Aus/Gewartet
 choice.showliquids=Flüssigkeiten
 choice.shifttoggledblock=Blöcke schleichend umschalten
 choice.shifttoggledents=Lebewesen schleichend umschalten
-choice.hidefromlist=Auf Spielerliste verstecken
+choice.hidefromlist=Verstecken vor der Spielerliste
+choice.hidefromdebug=Verstecken vor dem Bug-Menü
+choice.showitem=Gegenstand anzeigen
+choice.tts=Schildernde Blöcke
```

Comment:
The "TTS" stands for "Text to speech"...
```suggestion
choice.tts=Blöcke vorlesen
```

Reply:
Ok, I didn't know that.